python/python/actsvg/actsvg_json.py

```diff
@@ -0,0 +1,112 @@
+import json as jsn
+import actsvg
+from actsvg import proto
+
+""" read a grid axis from a json file"""
+
+
+@staticmethod
+def read_grid_axis(json_grid_axis):
+    edges = []
+    if json_grid_axis["type"] == 0:
+        n_bins = json_grid_axis["bins"]
+        a_range = json_grid_axis["range"]
+        for i in range(n_bins + 1):
+            edges.append(a_range[0] + i * (a_range[1] - a_range[0]) / n_bins)
+    return edges
+
+
+""" read a grid from a json file"""
+
+
+@staticmethod
+def read_grid(json_grid):
+    grid = proto.grid()
+    try:
+        grid_axes = json_grid["axes"]
+        grid.edges_0 = read_grid_axis(grid_axes[0])
+        if len(grid_axes) > 1:
+            grid.edges_1 = read_grid_axis(grid_axes[1])
+        index_data = json_grid["data"]
+
+    except KeyError:
+        print("** pyactsvg **: `read_grid` problem in key access ")
+    return grid, index_data
+
+
+""" read surface mateiral from a json file"""
+
+
+@staticmethod
+def read_surface_material(json_surface_material):
+    psm = proto.surface_material()
+    try:
+        surface_material = json_surface_material["value"]["material"]
+        surface_material_accessor = surface_material["accessor"]
+        # Convert the grid
+        accessor_grid = surface_material_accessor["grid"]
+        psm.grid, index_data = read_grid(accessor_grid)
+
+        # indicate the grid type
+        global_casts = surface_material["global_casts"]
+        if global_casts == ["binZ", "binPhi"]:
+            psm.grid.type = proto.grid.grid_type.z_phi
+            psm.grid.reference_r = 100
```

Comment:
Should this really be hardcoded?

Reply:
No, this should be read - will be fixed when 

https://github.com/acts-project/acts/pull/3039

is merged, and the output format updated.